### ISSUE TITLE
GitGraph is not rendering on VS Code v1.50.1 with WSL2

### ISSUE THREAD

**Issue by lesmo** [mhutchie/vscode-git-graph#405]
**Describe the Bug**
GitGraph is not rendering on VS Code v1.50.1 with WSL2. It worked before, but to be honest I'm not sure exactly if a VS Code update or GitGraph update broke it.

I tried to see if someone saw thisbefore, but these issues refer to earlier versions of both VS Code and GitGraph and thus they were already resolved:
* https://github.com/mhutchie/vscode-git-graph/issues/355
* https://github.com/microsoft/vscode/issues/101272

Moreover, these issues are related to some problems with resource importing that is clearly shown in the output panel... I'm unable to see any of that here.

**Steps to Reproduce**
Steps to reproduce the behaviour:
1. Having WSL2, open in VSCode a folder inside WSL2
2. Run the command to show GitGraph in Command Palette
3. See the GitGraph renders only "Repo" and "Branches" text, along with a "Show Remote Branches" checkbox

**Expected Behaviour**
GitGraph should be usable in VS Code.

**Environment**
 - Git Graph Extension Version: 1.26.0
 - Visual Studio Code Version: 1.50.1
 - Operating System: Windows 10 Pro
 - WSL2

VS Code info dump:
```
Version: 1.50.1 (user setup)
Commit: d2e414d9e4239a252d1ab117bd7067f125afd80a
Date: 2020-10-13T15:06:15.712Z
Electron: 9.2.1
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041
```

**Screenshots**
![image](https://user-images.githubusercontent.com/2228196/97486981-d8fce680-1921-11eb-83fb-a6757995ff43.png)

**Additional Context (optional)**
I'm using WSL2 with Ubuntu 20. The project folders are inside the Linux filesystem.

**Comment by mhutchie** [comment]
Hi @lesmo,

> Moreover, these issues are related to some problems with resource importing that is clearly shown in the output panel... I'm unable to see any of that here.

The easiest way to see the resource importing error logs is:
1. Have the Git Graph Webview visible.
2. Run the Visual Studio Code command "Developer: Open Webview Developer Tools".
3. Select the "Console" tab.

Please raise this as a bug in the Visual Studio Code repository, as this is definitely another Visual Studio Code Webview bug.

FYI: This year there have probably been close to 10 bugs affecting Visual Studio Code Webview's, introduced in different versions of Visual Studio Code. It's been extremely frustrating as there is nothing more I can do, as all extensions can do is provide the HTML, CSS & JavaScript for the Webview to use - Visual Studio Code is responsible for providing these required resources, and all other basic capabilities of the Webview. Not a single Webview bug this year has been an issue with Git Graph, they've all been caused by Visual Studio Code. This is partly because they way Git Graph interacts with the Visual Studio Code Webview API is extremely simple, and has only change once since Git Graph was created close to two years ago. In contrast, Microsoft has been doing significant overhauls of Webview's internally throughout this year.

**Comment by lesmo** [comment]
Well, thanks for your time.

**Comment by miltonjonat** [comment]
Hi there @lesmo. I've got the exact same behavior, also under WSL2. My VS Code, Electron, etc versions are *exactly* like yours (but the underlying distro is Ubuntu 18). Do you have any pointers to a related issue in the VS Code repo?

**Comment by lesmo** [comment]
@miltonjonat sadly no, I don't have any pointers... the problem got kinda solved on its own with v1.27.0 of GitGraph, same VSCode (v1.50.1) and same Ubuntu distro (mine's 20 though) seems to work with the new version.

No idea as to why.
